.gitlab-ci.yml

```diff
@@ -90,22 +97,42 @@ build:rust:linux:release:          &build
 
 
 
+build:rust:doc:release:            &build
+  stage:                           build
+  artifacts:
+    name:                          "${CI_JOB_NAME}_${CI_COMMIT_REF_NAME}-doc"
+    when:                          on_success
+    expire_in:                     7 days
+    paths:
+    - ./crate-docs
+  only:
+    - master
+    - tags
+    - web
+    - publish-rustdoc
+  tags:
+    - linux-docker
+  script:
+    - time cargo doc --release --verbose
+    - cp -R ./target/doc ./crate-docs
+    - echo "<meta http-equiv=refresh content=0;url=foo/index.html>" > ./crate-docs/index.html
```

Comment:
`foo`?

Reply:
yeah, wanted to replace it but current docs didn't compile well so I got distracted. What was your suggestion for the default landing page? @gnunicorn

Reply:
let's go with `substrate_service/index.html`